Sources/KIF/Additions/UIView-Debugging.m

```diff
@@ -136,6 +136,9 @@ - (void)printDatePickerState {
         case UIDatePickerModeCountDownTimer:
             printf(" UIDatePickerModeCountDownTimer");
             break;
+        case UIDatePickerModeYearAndMonth:
```

Comment:
I see we have a failure on CI due to this addition. It is not related to my functionality, I added it to make it compile while running tests locally, so it is weird 🤔
Should I remove it?

Reply:
Sorry, didn't get a chance to circle back around to this today. If I don't get time this weekend I'll set a reminder for Monday morning to check out the failure.

If you know it is related to the code you added and is tangential to the fix, it's up to you if you think the best path is removing it. I'll have to take a closer look.